backend/agentd/session.go

```diff
@@ -650,6 +650,8 @@ func (s *Session) handleEvent(_ context.Context, payload []byte) error {
 		} else {
 			eventBytesSummary.WithLabelValues(metrics.EventTypeLabelCheck).Observe(float64(len(payload)))
 		}
+		// TODO (ck) Should these be going through keepalived (TopicKeepalive)
+		// or straight to eventd (TopicKeepaliveRaw)?
```

Comment:
This should stay as it is. Let's amend the comment to mention that it's a compatibility fallback.

Reply:
Spent some time looking at the history here and decided to treat these like regular events as they were prior to 6.4.